src/app/common/menu/MenuList.jsx

```diff
@@ -42,11 +41,11 @@ const propTypes = {
 const generateMenuLinks = (links, location) => {
   const navLinks = !!links && links.map(link =>
     <List.Item
-      key={link.id}
+      key={link.path}
```

Comment:
Should these keys be different? Feels like on key could be the id, and the other could be the path.

Reply:
The id is no longer being generated. Before it was generated [here](https://github.com/cerner/terra-site/pull/13/files#diff-2e239fdb31751e56fd0b23b7a78e5d13L33) which just set the path as the id.